src/app/tests/suites/certification/ci-pics-values

```diff
@@ -2441,97 +2441,132 @@ REFALM.S.E00=1
-PIXIT.RVCCLEANM.MODE_CHANGE_FAIL=1
-PIXIT.RVCCLEANM.MODE_CHANGE_OK=1
-
-# RVC OPERATIONAL STATE CLUSTER
 RVCOPSTATE.S=1
-RVCOPSTATE.C=1
-
-#ManuallyControlled
-# These are the PICS supported by the all-clusters-app
-
-RVCOPSTATE.S.M.ST_STOPPED=1
-RVCOPSTATE.S.M.ST_RUNNING=1
-RVCOPSTATE.S.M.ST_PAUSED=1
-RVCOPSTATE.S.M.ST_ERROR=0
-RVCOPSTATE.S.M.ST_SEEKING_CHARGER=0
-RVCOPSTATE.S.M.ST_CHARGING=0
-RVCOPSTATE.S.M.ST_DOCKED=0
-RVCOPSTATE.S.M.ERR_NO_ERROR=1
-RVCOPSTATE.S.M.ERR_UNABLE_TO_START_OR_RESUME=0
-RVCOPSTATE.S.M.ERR_UNABLE_TO_COMPLETE_OPERATION=0
-RVCOPSTATE.S.M.ERR_COMMAND_INVALID_IN_STATE=0
-RVCOPSTATE.S.M.ERR_FAILED_TO_FIND_CHARGING_DOCK=0
-RVCOPSTATE.S.M.ERR_STUCK=0
-RVCOPSTATE.S.M.ERR_DUST_BIN_MISSING=0
-RVCOPSTATE.S.M.ERR_DUST_BIN_FULL=0
-RVCOPSTATE.S.M.ERR_WATER_TANK_EMPTY=0
-RVCOPSTATE.S.M.ERR_WATER_TANK_MISSING=0
-RVCOPSTATE.S.M.ERR_WATER_TANK_LID_OPEN=0
-RVCOPSTATE.S.M.ERR_MOP_CLEANING_PAD_MISSING=0
-
-#Events
-RVCOPSTATE.S.E00=1
-RVCOPSTATE.S.E01=1
-
-#Server
 RVCOPSTATE.S.A0000=1
 RVCOPSTATE.S.A0001=1
-RVCOPSTATE.S.A0002=1
+RVCOPSTATE.S.A0002=0
 RVCOPSTATE.S.A0003=1
 RVCOPSTATE.S.A0004=1
 RVCOPSTATE.S.A0005=1
-
-#Commands
+RVCOPSTATE.S.E00=1
+RVCOPSTATE.S.E01=1
 RVCOPSTATE.S.C00.Rsp=1
 RVCOPSTATE.S.C01.Rsp=0
 RVCOPSTATE.S.C02.Rsp=0
 RVCOPSTATE.S.C03.Rsp=1
 RVCOPSTATE.S.C04.Tx=1
 RVCOPSTATE.S.C128.Rsp=1
-RVCOPSTATE.C.C00.Tx=1
-RVCOPSTATE.C.C01.Tx=1
-RVCOPSTATE.C.C02.Tx=1
-RVCOPSTATE.C.C04.Tx=1
-
+RVCOPSTATE.S.M.ST_STOPPED=1
+RVCOPSTATE.S.M.ST_RUNNING=1
+RVCOPSTATE.S.M.ST_PAUSED=1
+RVCOPSTATE.S.M.ST_ERROR=1
+RVCOPSTATE.S.M.ST_SEEKING_CHARGER=1
+RVCOPSTATE.S.M.ST_CHARGING=1
+RVCOPSTATE.S.M.ST_DOCKED=1
+RVCOPSTATE.S.M.ERR_NO_ERROR=1
+RVCOPSTATE.S.M.ERR_UNABLE_TO_START_OR_RESUME=1
+RVCOPSTATE.S.M.ERR_UNABLE_TO_COMPLETE_OPERATION=1
+RVCOPSTATE.S.M.ERR_COMMAND_INVALID_IN_STATE=1
+RVCOPSTATE.S.M.ERR_FAILED_TO_FIND_CHARGING_DOCK=1
+RVCOPSTATE.S.M.ERR_STUCK=1
+RVCOPSTATE.S.M.ERR_DUST_BIN_MISSING=1
+RVCOPSTATE.S.M.ERR_DUST_BIN_FULL=1
+RVCOPSTATE.S.M.ERR_WATER_TANK_EMPTY=1
+RVCOPSTATE.S.M.ERR_WATER_TANK_MISSING=1
+RVCOPSTATE.S.M.ERR_WATER_TANK_LID_OPEN=1
+RVCOPSTATE.S.M.ERR_MOP_CLEANING_PAD_MISSING=1
+RVCOPSTATE.C=0
+RVCOPSTATE.C.C00.Tx=0
+RVCOPSTATE.C.C01.Tx=0
+RVCOPSTATE.C.C02.Tx=0
+RVCOPSTATE.C.C04.Tx=0
 
-# RVC RUN MODE CLUSTER
 RVCRUNM.S=1
-RVCRUNM.S.F00=0
-
-#Server
 RVCRUNM.S.A0000=1
 RVCRUNM.S.A0001=1
-
-#Commands
 RVCRUNM.S.C00.Rsp=1
 RVCRUNM.S.C01.Tx=1
-
+RVCRUNM.S.F00=0
 RVCRUNM.S.M.CAN_TEST_MODE_FAILURE=1
 RVCRUNM.S.M.CAN_MANUALLY_CONTROLLED=1
 
-#PIXIT
-PIXIT.RVCRUNM.MODE_CHANGE_FAIL=1
-PIXIT.RVCRUNM.MODE_CHANGE_OK=1
+# These are useless as these values are set where the python tests are run.
+#  They are only here because a list test wants then to be.
+PIXIT.RVCCLEANM.MODE_CHANGE_FAIL=0
+PIXIT.RVCCLEANM.MODE_CHANGE_OK=0
+PIXIT.RVCRUNM.MODE_CHANGE_FAIL=0
+PIXIT.RVCRUNM.MODE_CHANGE_OK=0
+
+# These are the RVC PICS for the all-clustrs-app.
+# These PICS are commented out because the yaml tests are being run against the rvc-app.
```

Comment:
Should these be removed?

Reply:
Eventually yes. I think the idea going forward is to not have this file altogether and instead have separated files for the different apps. I kept this here so when that time comes it will be easier populate the PICS file for the all-clusters-app.